patches/chrome-browser-media-router-media_router_feature.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/media/router/media_router_feature.cc b/chrome/browser/media/router/media_router_feature.cc
+index 9c42d30ecd22335ff314d366d2729fc6f4af3241..9470174961e1b01eba328d14e1396232b5425d5c 100644
+--- a/chrome/browser/media/router/media_router_feature.cc
++++ b/chrome/browser/media/router/media_router_feature.cc
+@@ -55,6 +55,7 @@ const PrefService::Preference* GetMediaRouterPref(
+ #endif  // defined(OS_ANDROID) || BUILDFLAG(ENABLE_EXTENSIONS)
+ 
+ bool MediaRouterEnabled(content::BrowserContext* context) {
++  ReturnFalseIfMediaRouterDisabled();
```

Comment:
this doesn't need a patch at all, you can do a chromium_src override for the method